html/user-profile.html

```diff
@@ -5,35 +5,56 @@
     <meta content="text/html; charset=UTF-8">
     <link rel="stylesheet" type="text/css" href="../css/style.css" />
 </head>
-<body>
-  <header>
-    <nav>
-      <ul>
-        <li><a href="user-profile.html">User Profile</a></li>
-        <li><a href="user-list.html">User List</a></li>
-        <li><a href="create-user.html">Create User</a></li>
-        <li><a href="edit-user.html">Edit User</a></li>
-      </ul>
-    </nav>
-  </header>
-  <table> 
-      <h3>User Profile</h3>
-    <tr>
-      <td>ID:</td>
-      <td>#0000</td>
-    </tr>
-    <tr>
-      <td>Name:</td>
-      <td>John Smith</td>
-    </tr>
-    <tr>
-      <td>Phone:</td>
-      <td>000-000-0000</td>
-    </tr>
-    <tr>
-      <td>Email:</td>
-      <td>john.smith@banno.com</td>
-    </tr>
-  </table>
-</body>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="user-profile.html">User Profile</a></li>
+          <li><a href="user-list.html">User List</a></li>
+          <li><a href="create-user.html">Create User</a></li>
+          <li><a href="edit-user.html">Edit User</a></li>
+        </ul>
+      </nav>
+    </header>
+    <h3>User Profile</h3>
+    <table> 
+      <tr>
+        <td>ID:</td>
+        <td>#0000</td>
+      </tr>
+      <tr>
+        <td>Name:</td>
+        <td>John Smith</td>
+      </tr>
+      <tr class="expand">
+        <td>Phone:</td>
+        <td>000-576-5309</td>
+      </tr>
+      <tr class="expand">
+        <td>Email:</td>
+        <td>john.smith@banno.com</td>
+      </tr>
+      <tr>
+        <td>
+          <button id="bText"  Onclick="openClose()">Collapse</button>
+        </td>
+      </tr>
+    </table>
+    <script>
+      function openClose() {
+        var collapse = document.getElementsByClassName("expand");
+        var buttonText = document.getElementById("bText");
+
+        for (var i = 0; i < collapse.length; i++) {
+          collapse[i].classList.toggle("collapse");
+        }
+
+        if (buttonText.innerHTML === "Collapse") {
```

Comment:
This would be a good spot to try out a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

Reply:
I'll mess around with this too and see if I can make it work. I'll let you know if I have questions.